### DISCUSSION TITLE
How to test functionality

### DISCUSSION THREAD

**Discussion by skyfall17** [jef/streetmerchant#934] [Help]
### Description

How do I test the program for a specific series?

Would I type _SHOW_ONLY_SERIES=test:3070_ in .env?

If not, how can I test that a website is opened upon finding a product?

**Comment by lchilson** [comment]
that would be awesome if we could have a quick intro to adding an item so we could see how to test this

**Comment by jef** [comment]
Exactly. Testing using `SHOW_ONLY_SERIES=test:series`

> that would be awesome if we could have a quick intro to adding an item so we could see how to test this

Not bad idea. Perhaps I'll add to the doc. Thanks!

**Reply by vincefrietema** [reply]
yo jef do i need to set my command prompt every time before i leave and enter the bot

**Reply by Mr-Tech-13** [reply]
Leave and enter?
